examples/group/README.md

```diff
@@ -0,0 +1,182 @@
+The setup target `setup-lab` of the `Makefile` in `pachyderm/examples/group` will create a directory (labresults) with our example data.
+In the `examples/group` directory, run:
+```shell
+$ make setup-lab
+```
+***Step 2*** - Create/populate Pachyderm's repository and create your pipelines
+In the `examples/joins` directory, run:
+```shell
+$ make deploy-lab
+```
+or run:
+```shell
+$ pachctl create repo labresults
+$ pachctl put file -r labresults@master:/ -f labresults
+$ pachctl create pipeline -f lab_group_by_hospital.json 
+$ pachctl create pipeline -f lab_group_by_patient.json
+```
+Have a quick look at your repositories: 
+```shell
+$ pachctl list file labresults@master
+```
+You should see the following files:
+
+![labresults repo](./img/list_file_labresults_master.png)
+
+The commit in your entry repository has triggered the execution of your pipeline (i.e., a job). Have a quick check at your pipeline's status:
+```shell
+$ pachctl list pipeline
+```
+Once it has run successfully, you should see something like this:
+
+![pipelines](./img/lab_list_pipeline.png)
+
+***Step 3*** - Let's have a look at our final product: Check the output repository of your pipeline.
+```shell
+$ pachctl list file group_by_patient@master
+$ pachctl list file group_by_hospital@master
+```
+You should see your expected sub repositories. 
+
+Check one testresult for patient 1:
+```shell
+$ pachctl get file group_by_patient@master:/1/T1606707613-LIPID-PATID1-CLIA24D9871328.txt
+```
+
+## 4. Example 2 - Group pipeline on several repositories 
+***Goal***
+For each store, we are going to calculate the net amount of all transactions (net_amount = order_total - return_total) and save it to a text file named after the store identifier.
+
+1. **Pipeline input repositories**: `stores` ,`returns`, `purchases` - Group by STOREID on all 3 repositories. Each match (i.e., all transactions (purchases and returns) having occured at a given store along with the store information itself) will generate a datum.
+2. **Pipeline**: Executes a python code reading the `purchases` and `returns` for each matching STOREID and writing the corresponding net_amount to a text file named after the STOREID. 
+3. **Pipeline output repository**: `group_store_revenue`- list of text files named after the STOREID. 
+
+In the diagram below, we have mapped out our data. 
+
+![group_example2](./img/group_example2.png)
+
+The following table lists the expected result for each store. 
+
+![group_example2_digest](./img/group_example2_digest.png)
+
+***Step 4*** - Let's create your new data.
+In the `examples/group` directory, run:
+```shell
+$ make setup-retail
+```
+You just created 3 directories: stores, purchases, returns. Check them out.
+```shell
+$ ls ./purchases
+```
+***Step 5*** - Create/populate Pachyderm’s repository and create your pipelines
+In the `examples/group` directory, run:
+```shell
+$ make deploy-retail
```

Comment:
This target failed on 1.12.0-6ac9505faae8ca32341ada7bf40ab94f238a4bd6.  I will try it again on RC-2.

Reply:
Please do. The patch fixing the group features was not included in the rc1, hence the rc2. Depending on when you packaged your 1.12 version on master, you might not have the full group feature.